App.js

```diff
@@ -1,20 +1,66 @@
+import { useCallback } from 'react';
 import { StatusBar } from 'expo-status-bar';
 import { LogBox } from 'react-native';
-import { NativeBaseProvider } from 'native-base';
+import { NativeBaseProvider, extendTheme } from 'native-base';
+import { useFonts } from 'expo-font';
+import * as SplashScreen from 'expo-splash-screen';
 
 import { AuthContextProvider } from './contexts/AuthContext';
 import { NotificationContextProvider } from './contexts/NotificationContext';
 import { ChatContextProvider } from './contexts/ChatContext';
 import AppStack from './stacks/AppStack';
 
+// SplashScreen.preventAutoHideAsync();
+
 export default function App() {
   LogBox.ignoreAllLogs();
+  const [fontsLoaded] = useFonts({
+    'SofiaPro-Black': require('./assets/fonts/SofiaPro-Black.otf'),
+    'SofiaPro-Bold': require('./assets/fonts/SofiaPro-Bold.otf'),
+    'SofiaPro-Light': require('./assets/fonts/SofiaPro-Light.otf'),
+    'SofiaPro-Medium': require('./assets/fonts/SofiaPro-Medium.otf'),
+    'SofiaPro-Regular': require('./assets/fonts/SofiaPro-Regular.otf'),
+    'SofiaPro-SemiBold': require('./assets/fonts/SofiaPro-SemiBold.otf'),
+    Quicksand: require('./assets/fonts/Quicksand-Bold.ttf'),
+  });
+
+  console.log(fontsLoaded);
+
+  const onLayoutRootView = useCallback(async () => {
+    if (fontsLoaded) {
+      await SplashScreen.hideAsync();
+    }
+  }, [fontsLoaded]);
+
+  if (!fontsLoaded) {
+    return null;
+  }
+
+  const theme = extendTheme({
```

Comment:
Please remind to fix this on friday. Just a small improvement what we can do is we can keep App.js file clear just have imports in here and move them to Another file and export them. We can try to do it...